package.json

```diff
@@ -7,19 +7,19 @@
     "bugs": "https://github.com/semaphore-protocol/semaphore/issues",
     "private": true,
     "scripts": {
-        "build": "yarn build:libraries && yarn build:subgraph",
-        "build:libraries": "yarn workspaces foreach -A -t --no-private run build",
-        "build:subgraph": "yarn workspace semaphore-subgraph codegen sepolia && yarn workspace semaphore-subgraph build",
+        "build": "yarn workspaces foreach -Ap --topological-dev run build",
+        "build:libraries": "yarn workspaces foreach -Apt --no-private run build",
+        "build:subgraph": "yarn workspace semaphore-subgraph build",
         "compile:contracts": "yarn workspace semaphore-contracts compile",
-        "test": "yarn test:libraries && yarn test:contracts && yarn test:circuits && yarn test:subgraph",
+        "test": "concurrently -g -c auto \"yarn:test:*\"",
         "test:libraries": "jest --coverage",
-        "test:library": "jest packages/${0}",
+        "test.library": "jest packages/${0}",
```

Comment:
Maybe a bit awkward, this is to avoid catching `test:library` when using the wildcard for the `test` script:
```
concurrently -g -c auto yarn:test:*
```

Reply:
Otherwise we can explicitly list all the commands

Reply:
I'd prefer keeping the same format for all commands with `:`, so maybe better to list all of them (they'll always be just three in the end).